### ISSUE TITLE
Exhentai & Chaika namespace tags (female/male) in metadata?

### ISSUE THREAD

**Issue by UnlimitedArk** [HDoujinDownloader/HDoujinDownloader#369]
Is there any way for hdoujin dl to actually grab the namespaces "female, male" etc when downloading from exh or chaika?
As i have set it up for the metadata info.txt, the only namespaces it grabs are artist, parody, circle and such, but all the other tags are without namespace, was wondering if there was a way to set it up properly.
Also unrelated but was wondering if the tags could be grabbed in lowercase? I usually correct it manually so it's no biggie but would be nice if i could set it lowercase as default.
Thank you very much

**Comment by HDoujinDownloader** [comment]
I updated the module for Chaika to honor tag namespaces, so they should appear the way you're expecting now. They should already be working for ExHentai, though. Is there a particular gallery they're not being parsed from properly? 

Just to be sure: Under _**Settings > Save to > Metadata**_, click "Metadata file settings", and ensure that the "Strip tag namespaces" setting is disabled.

I've gotten several comments over the years about making tags lowercase, and I've now decided to make them lowercase by default. They should be good to go without any action on your part after the next program update, and there will be a new "Metadata.General.TagCase" under **_Tools > General settings > Advanced settings_** for restoring the previous behavior.

Thank you for being so patient!
